scaaml/capture/input_generators/attack_point_iterator.py

```diff
@@ -27,11 +29,19 @@ def __init__(self, configuration) -> None:
         """Initialize a new iterator."""
         self._attack_point_iterator_internal: AttackPointIteratorInternalBase
         if configuration["operation"] == "constants":
-            constant_iterator = AttackPointIteratorInternalConstants(
+            constant_iter = AttackPointIteratorInternalConstants(
                 name=configuration["name"],
                 values=configuration["values"],
             )
-            self._attack_point_iterator_internal = constant_iterator
+            self._attack_point_iterator_internal = constant_iter
+        elif configuration["operation"] == "balanced_generator":
```

Comment:
See comment in the DD an alternative to `if / elif / else` which could make things more readable.

Reply:
I will implement this in a later PR